entity-types/infra-elasticsearchnode/definition.yml

```diff
@@ -1,5 +1,43 @@
 domain: INFRA
 type: ELASTICSEARCHNODE
+
+synthesis:
+  # Apart from this rule ELASTICSEARCH entities are also created from infra pipeline.
+  rules:
+  - compositeIdentifier:
+      separator: ":"
+      attributes:
+        - elasticsearch.cluster.name
+        - elasticsearch.node.name
+    name: elasticsearch.node.name
+    encodeIdentifierInGUID: true
+    conditions:
+      - attribute: eventType
+        value: Metric
+      # All metrics from the receiver starts with the 'elasticsearch.' prefix.
+      - attribute: metricName
+        prefix: elasticsearch.
+      - attribute: instrumentation.provider
+        value: opentelemetry
+      # This filters only metrics coming from the elasticsearchreceiver, given that 
+      # metrics could differ between different runtime receivers. 
+      - attribute: otel.library.name
+        value: otelcol/elasticsearchreceiver
+    tags:
+      # Default resource attributes
+      elasticsearch.node.name:
+      elasticsearch.cluster.name:
+      # OTel attributes
+      # The library name contains the name of the receiver that is used to identify the source
+      # and select the dashboard.
+      otel.library.name:
+        entityTagName: instrumentation.name
+
+  tags:
```

Comment:
should we add some of them in the Golden Tags below?

Reply:
Do you mean adding these two to `goldenTags`?:

- elasticsearch.cluster.name
- elasticsearch.node.name

Reply:
@paologallinaharbur Just wanted to check back with you on this question ^^

Reply:
yes it would make sense